MOBILE_CLIENT/ANDROID/_MEDIUM/IMPLICIT_PENDINGINTENT/context.md

```diff
@@ -0,0 +1,84 @@
+`PendingIntent` in Android is indeed a powerful class, and with great power comes the need for careful consideration. Here are the main concerns you should be aware of when using `PendingIntent`:
```

Comment:
This is not a context section. Context has 0 technical details, it is about real compromise cases and non technical impact.